rascal-lsp/src/main/java/org/rascalmpl/vscode/lsp/parametric/model/ParametricSummaryBridge.java

```diff
@@ -195,6 +199,10 @@ public InterruptibleFuture<T> lookup(Position cursor) {
         }
     }
 
+    @FunctionalInterface
+    public interface SummaryCalculator {
```

Comment:
NP: could also just extend `BiFunction`? or maybe use that in the constructor (at the cost of a very verbose syntax since java is lacking aliassing feature)